app/component/itinerary/Itinerary.js

```diff
@@ -1019,6 +1035,21 @@ const containerComponent = createFragmentContainer(ItineraryWithBreakpoint, {
             }
             network
           }
+          rentalVehicle {
+            vehicleId
+            name
+            lat
+            lon
+            network
+            rentalUris {
+              android
+              ios
+              web
+            }
+            vehicleRentalSystem {
```

Comment:
Are these new request fields used? Request shoudl be always minimal for each component level.

Reply:
Good point! Forgot to tidy this up.